RemoveNthNodeFromEnd.java

```diff
@@ -0,0 +1,54 @@
+package LinkedLists;
+/**
+ * Time omplexity: O(n)
+ * Space complexity: O(1)
+ * Leetcode accepted: yes
+ * Problems faced: Tried two-pointer approach, but missed edge conditions in lines 41 & 43 initially
```

Comment:
try considering the edge cases prior to implementing the solution and after the amount of practice we are doing it will not be an issue anymore